### PR TITLE
USF-1474: Add Storefront Compatibility package doc

### PR DESCRIPTION
**Pull request by keharper**
This PR adds installation information about the storefront compatibility package.

### CODE REVIEW

File: src/content/docs/get-started/storefront-compatibility.mdx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+import Aside from '@components/Aside.astro';
+import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
+
+
+The Storefront Compatibility Package contains changes to the Adobe Commerce 2.4.7 codebase that enable dropin functionality. Most of these changes enhance the Adobe Commerce GraphQL schema. However, miscellaneous bugfixes are also included.
+
+## Release information
+
+**Version:** 4.7.0-beta14
+**Release date:** TBD
```
Comment (sirugh) [inline src/content/docs/get-started/storefront-compatibility.mdx]: Technically already "released" to repo/connect20 although I think only accessible via MAGE_ID

File: src/content/docs/get-started/storefront-compatibility.mdx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+* [`generateCustomerToken` mutation](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/generate-token/) - Creates a new customer token.
+
+* [`recaptchaFormConfig` query](https://developer.adobe.com/commerce/webapi/graphql/schema/store/queries/recaptcha-form-config/) - Returns information about the reCaptcha configuration for the specified form type.
+
+* `resendConfirmationEmail` mutation - Resends a confirmation email to the specified customer upon request.
+
+## Installation
+
+Partners can request access to the Storefront Compatibility Package by messaging us on the **TBD** Adobe Commerce internal Slack channel.
```
Comment (tkacheva) [inline src/content/docs/get-started/storefront-compatibility.mdx]: Adobe Commerce Enterprise Support Channel https://adobe-dx-support.enterprise.slack.com/archives/C07K42W9YB0

File: src/content/docs/get-started/storefront-compatibility.mdx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+---
+
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+import Callouts from '@components/Callouts.astro';
+import Aside from '@components/Aside.astro';
+import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
+
+
+The Storefront Compatibility Package contains changes to the Adobe Commerce 2.4.7 codebase that enable dropin functionality. Most of these changes enhance the Adobe Commerce GraphQL schema. However, miscellaneous bugfixes are also included.
```
Comment (tkacheva) [inline src/content/docs/get-started/storefront-compatibility.mdx]: call out that core commerce dropins that power low funnel experience (Cart/checkout/Accoutn/ Auth) require SCP

**Review by jeff-matthews — APPROVED** [review]

File: src/content/docs/get-started/storefront-compatibility.mdx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+1. Use Composer to add the package module to your project:
+
+   ```bash
+   composer require adobe-commerce/storefront-compatibility
+   ```
+
+1. Update dependencies and install the extension:
+
+   ```bash
+   composer update  "adobe-commerce/storefront-compatibility"
```
Comment (sivaschenko) [inline src/content/docs/get-started/storefront-compatibility.mdx]: I don't think it makes sense to run `update  "adobe-commerce/storefront-compatibility"` after `require adobe-commerce/storefront-compatibility` as require is installing the latest available package version

File: src/content/docs/get-started/storefront-compatibility.mdx [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+1. Upgrade Adobe Commerce:
+
+   ```bash
+   bin/magento setup:upgrade
+   ```
+
+1. Clear the cache:
+
+   ```bash
+   bin/magento cache:clean
```
Comment (sivaschenko) [inline src/content/docs/get-started/storefront-compatibility.mdx]: I don't think `cache:clean` is necessary to run after `setup:upgrade` as the cache is cleaned as part of upgrade command.

**Review by sivaschenko — COMMENTED** [review]
Please see my 2 suggestions to simplify installation steps

**Review by sivaschenko — APPROVED** [review]

File: src/content/docs/get-started/requirements.mdx [hunk 6]
```diff
@@ -30,6 +30,14 @@ The following links provide the requirements and installation procedures for you
 
 <Task>
 
+### Storefront Compatibility package (optional)
+
+The [Storefront Compatibility package](https://experienceleague.adobe.com/developer/commerce/storefront/get-started/launch-checklist/storefront-compatibility/) extends the GraphQL schema provided with Adobe Commerce. It provides several new mutations and adds missing fields that are needed to fully implement your dropins.
```
Comment (dshevtsov) [inline src/content/docs/get-started/requirements.mdx:35]: The links returns 404

Comment (jeff-matthews) [reply src/content/docs/get-started/requirements.mdx:35]: @dshevtsov, Kevin created a PR to revert this (#50) but it hasn't been merged yet.

I would disregard the Jenkins prod build.

**Pull request merged**